### ISSUE TITLE
[Mailer] \Symfony\Component\Mailer\Event\MessageEvent never consumed

### ISSUE THREAD

**Issue by Perf** [symfony/symfony#31449]
**Symfony version(s) affected**: 4.3.0-BETA1

**Description**  
When sending email using a new `symfony/mailer` component an event `\Symfony\Component\Mailer\Event\MessageEvent` us dispatched.
Mailer component has two subscribers for this event:
`\Symfony\Component\Mailer\EventListener\MessageListener`
`\Symfony\Component\Mailer\EventListener\EnvelopeListener`
but non of them are fired in case of an event.

Command `bin/console de:ev` also doesn't shows these listeners.

**How to reproduce**  
1. create one simple twig template for html part of email, for example:
```
# templates/email/hello.html.twig
<p>Hi {{ email.to }}!</p>
```
2. create a console command and inject Mailer in constructor
```php
    private $mailer;

    public function __construct(MailerInterface $mailer)
    {
        $this->mailer = $mailer;
        parent::__construct();
    }
```
3. put in `execute` method something like:
```php
        $email = new TemplatedEmail();
        $email
            ->from('no-reply@domain.com')
            ->to('me@domain.com')
            ->subject('Test email message')
            ->htmlTemplate('email/hello.html.twig')
            ;

        $this->mailer->send($email);
```
3. get an Exception
> In HandleMessageMiddleware.php line 82:
> A message must have a text and/or an HTML part.
> In Email.php line 427:
> A message must have a text and/or an HTML part.

This exception raised because `\Symfony\Component\Mailer\EventListener\MessageListener` was not fired.

**Possible Solution**  
¯\\\_(ツ)\_/¯

**Additional context**  
`symfony/messenger` is installed with default configuration
`symfony/swiftmailer-bundle` is not installed

**Comment by Perf** [comment]
Update:
after adding following lines in `services.yaml`
```yaml
    Symfony\Component\Mailer\EventListener\MessageListener:
        tags: ['kernel.event_subscriber']

    Symfony\Component\Mailer\EventListener\EnvelopeListener:
        tags: ['kernel.event_subscriber']
```
For event `MessageEvent` both subscribers are fired to process it, BUT when `MessageListener` is created, an instance of `BodyRenderer` is not injected.

**Comment by Perf** [comment]
Update2:
explicitly configuring `$renderer` argument helps... but all this issue looks like services autoconfigure and autowire is not properly working, I guess.
```yaml
    Symfony\Component\Mailer\EventListener\MessageListener:
        tags: ['kernel.event_subscriber']
        arguments:
            $renderer: '@twig.mime_body_renderer'
```

**Comment by fabpot** [comment]
@Perf fixed in #31458

**Comment by bellu** [comment]
I'm using 5.4 LTS and this event is never fired.
